src/backend/wiki_game_info/protondb/utils.ts

```diff
@@ -12,7 +12,7 @@ export async function getInfoFromProtonDB(
   const url = `https://www.protondb.com/api/v1/reports/summaries/${steamID}.json`
 
   const response = await axios
-    .get<ProtonDBInfo>(url, { headers: {} })
```

Comment:
I don't have knowledge about axios with types. My question in the first place was if it crashes if the response does not match the type or if it just ignores not fitting json values? Maybe you have experience here you can share?

Reply:
I just made it non-typed, it's much safer, and it handles a case where the field we want doesn't exist.